backend/app/controllers/users.controller.js

```diff
@@ -1,11 +1,14 @@
 const user = require('../models/users.model');
+let passwordHash = require('password-hash')
```

Comment:
I would recommend favouring const vs let. 
```suggestion
const passwordHash = require('password-hash')
```